customer.rb

```diff
@@ -0,0 +1,149 @@
+require_relative '../Bank/validetion.rb'
+require_relative '../Bank/account.rb'
+require_relative '../Bank/transaction.rb'
+
+
+class Customer 
+
+  @@status = 0 
```

Comment:
instead of using a class variable, create a new class 'Database' and use the singleton method.

Reply:
Use this Database class to handle transactions, accounts, and bank balance.